pkgs/top-level/all-packages.nix

```diff
@@ -8651,6 +8651,11 @@ with pkgs;
     inherit (linuxPackages) nvidia_x11;
     nvidiaGpuSupport = config.cudaSupport or false;
   };
+  nomad_1_3 = callPackage ../applications/networking/cluster/nomad/1.3.nix {
+    buildGoModule = buildGo118Module;
```

Comment:
I was surprised by this change, double checking nomad 1.3 is still on go 1.17, see e.g. https://github.com/hashicorp/nomad/blob/v1.3.0/.go-version or CI pipelines. No mention in the changelog either.